src/xbot/common/subsystems/pose/BasePoseSubsystem.java

```diff
@@ -14,6 +15,7 @@
 public abstract class BasePoseSubsystem extends BaseSubsystem implements PeriodicDataSource {
 
     public final XGyro imu;
+    AHRS ahrs;
```

Comment:
You already have the `public final XGyro imu`. Since you added the angular velocity method in the XGyro interface, you can just call that, rather than creating another AHRS and calling it.